### PR TITLE
[PATCH] Minimize bundle size

### PR DESCRIPTION
**Pull request by maxnachlinger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ curl https://demo.ckan.org/api/3/action/package_show?id=adur_district_spending >
 cat input.json | npx json-to-simple-graphql-schema > output.graphql
 ```
 
+### Usage in the browser:
```
Comment (maxnachlinger) [inline README.md:33]: Yup this works :) I've tried it.

File: package.json [hunk 2]
```diff
@@ -28,16 +28,21 @@
   },
   "dependencies": {
     "get-stdin": "6.0.0",
```
Comment (maxnachlinger) [inline package.json]: Using these really cuts the front-end bundle size down.

File: package.json [hunk 3]
```diff
@@ -28,16 +28,21 @@
...
+    "lodash.flow": "3.5.0",
+    "lodash.isboolean": "3.0.3",
+    "lodash.isinteger": "4.0.4",
+    "lodash.isnumber": "3.0.3",
+    "lodash.isobject": "3.0.2",
+    "lodash.set": "4.3.2",
     "pascal-case": "2.0.1"
   },
   "devDependencies": {
     "babel-eslint": "10.0.1",
```
Comment (maxnachlinger) [inline package.json:38]: Updated a few dependencies.

File: README.md [hunk 4]
```diff
@@ -17,19 +17,27 @@ npm i -g @walmartlabs/json-to-simple-graphql-schema
 
 Pipe in some JSON. Here's a cURL JSON response piped in to this app:
 ```bash
```
Comment (maxnachlinger) [inline README.md:19]: Much larger and more reliable data set

File: app.js [hunk 5]
```diff
@@ -17,11 +17,11 @@
 
 "use strict";
 
-const getStdin = require("get-stdin");
+const fs = require("fs");
```
Comment (maxnachlinger) [inline app.js:20]: Since this entry point is for a command line app, I'm fine with using `fs` and `readFileSync` on `0` - aka `stdin`

**Comment by maxnachlinger** [comment]
Made a simple front-end project to test these changes:
bundle size when using the master branch: 109 KiB
bundle size when using this branch: 45.6 KiB

**Review by ChrisCrewdson — APPROVED** [review]
Looks good to me!

**Pull request merged**